src/ApiPort.VisualStudio.Common/AssemblyFile.cs

```diff
@@ -51,8 +51,7 @@ public int Compare(IAssemblyFile x, IAssemblyFile y)
                 return y == null ? 0 : -1;
             }
 
-
-            return x.Name.CompareTo(y?.Name);
+            return string.CompareOrdinal(x.Name, y?.Name);
```

Comment:
cool! I've always done `StringComparer.Ordinal.CompareTo(x,y)` - this API is half the length :)